### PR TITLE
Fix links to issue tags in the contributing docs

### PR DESCRIPTION
**Pull request by gabriellacerda**
#hacktoberfest, thanks!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -55,35 +55,35 @@ Things that appear to be broken or are not working as intended.
 
 An enhancement to an existing feature.
 
-### [`external dependency` ![Issues tagged with 'external dependency'](https://img.shields.io/github/issues-raw/nasa-gibs/worldview/external%20dependency.svg)](https://github.com/nasa-gibs/worldview/issues?q=is%3Aopen+is%3Aissue+label%3Aexternal%20dependency)
+### [`external dependency` ![Issues tagged with 'external dependency'](https://img.shields.io/github/issues-raw/nasa-gibs/worldview/external%20dependency.svg)](https://github.com/nasa-gibs/worldview/issues?q=is%3Aopen%20is%3Aissue%20label%3A"external%20dependency")
 
 Issues that are waiting on something out of our control.
 
-### [`help wanted` ![Issues tagged with 'help wanted'](https://img.shields.io/github/issues-raw/nasa-gibs/worldview/help%20wanted.svg)](https://github.com/nasa-gibs/worldview/issues?q=is%3Aopen+is%3Aissue+label%3Ahelp%20wanted)
+### [`help wanted` ![Issues tagged with 'help wanted'](https://img.shields.io/github/issues-raw/nasa-gibs/worldview/help%20wanted.svg)](https://github.com/nasa-gibs/worldview/issues?q=is%3Aopen+is%3Aissue+label%3A"help%20wanted")
```
Comment (ZachTRice) [inline CONTRIBUTING.md]: Looking good. Could you also update the svg tags to include quotes?
i.e. https://img.shields.io/github/issues-raw/nasa-gibs/worldview/"help%20wanted".svg

Comment (gabriellacerda) [reply CONTRIBUTING.md]: Ah, ok

Comment (localjo) [reply CONTRIBUTING.md]: Make sure you encode those quotes with `%22` :)

Comment (ZachTRice) [reply CONTRIBUTING.md]: Good catch @localjo, 
the svg's would be: https://img.shields.io/github/issues-raw/nasa-gibs/worldview/%22help%20wanted%22.svg

and the links would be https://github.com/nasa-gibs/worldview/issues?q=is%3Aopen+is%3Aissue+label%3A%22help%20wanted%22

Comment (gabriellacerda) [reply CONTRIBUTING.md]: Shure, fixed

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -55,35 +55,35 @@ Things that appear to be broken or are not working as intended.
...
 
 These are new features to be developed at some point in the future.
 
-### [`ready for development` ![Issues tagged with 'ready for development'](https://img.shields.io/github/issues-raw/nasa-gibs/worldview/ready%20for%20development.svg)](https://github.com/nasa-gibs/worldview/issues?q=is%3Aopen+is%3Aissue+label%3Aready%20for%20development)
+### [`ready for development` ![Issues tagged with 'ready for development'](https://img.shields.io/github/issues-raw/nasa-gibs/worldview/%22ready%20for%20development%22.svg)](https://github.com/nasa-gibs/worldview/issues?q=is%3Aopen+is%3Aissue+label%3A%22ready%20for%20development%22)
 
 These are issues that are "on deck" for development. We're planning to work on these next.
 
-### [`ready for test` ![Issues tagged with 'ready for test'](https://img.shields.io/github/issues-raw/nasa-gibs/worldview/ready%20for%20test.svg)](https://github.com/nasa-gibs/worldview/issues?q=is%3Aopen+is%3Aissue+label%3Aready%20for%20test)
+### [`Testing` ![Issues tagged with 'testing'](https://img.shields.io/github/issues-raw/nasa-gibs/worldview/testing.svg)](https://github.com/nasa-gibs/worldview/issues?q=is%3Aopen+is%3Aissue+label%3Atesting)
```
Comment (ZachTRice) [inline CONTRIBUTING.md]: One more nitpicky change. Can you make `Testing` lowercase? Otherwise this looks solid 👍

Comment (gabriellacerda) [reply CONTRIBUTING.md]: No problem

**Comment by gabriellacerda** [comment]
Shure @localjo

**Comment by Benjaki2** [comment]
Valeu @gabriellacerda !

**Pull request merged**